cmake/IWYU.cmake

```
@@ -0,0 +1,36 @@
+find_program(IWYU_PATH iwyu_tool)
```

```suggestion
find_program(IWYU_PATH NAMES iwyu_tool iwyu-tool)
```

On Arch Linux the tool is called `iwyu-tool`